### DISCUSSION TITLE
Welcome to Tasmota_KNX Discussions!

### DISCUSSION THREAD

**Discussion by ascillato** [ascillato/Tasmota_KNX#566] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by detxm80** [comment]
Hi adrian, thanks for your works! Sorry for my bad english.
i've a sonoff 4ch pro that i use to turn on 4 lights and i've a knx system with a knx ip router that i use normally with home assistant.
i woult turn on light 3 on sonoff by a push button knx, i try to read your docs without success.
I ve configured push button with this group address:

3/0/14 ---> status notification and on/off

on sonoff i configured 1.1.36 (free address).

and then? 
i try various mode, on send command i've output (1 2 3 4), button (1 2 3 4), knx tx (1 2 3 4 5) and scene tx

on receive command i've output (1 2 3 4) output toggle (1 2 3 4) knx rx (1 2 3 4 5) and scene rx.

thanks for your help!

**Reply by ascillato** [reply]
Hi, sorry I don't quite understand how is your setup.

Do you have a button in Home assistant? or the button is another KNX device? or is a local button of your sonoff 4CH?

Can you post an screenshot of your KNX configuration in Tasmota?
Can you post the console log when you try that?

**Reply by detxm80** [reply]
I adrian, button is a knx device that i would to turn on position 3 of sonoff. 

Group address of knx button device is 3/0/14 that include on off and status notification
![4F8143FF-6C1E-4B89-B165-040A38807BFA](https://user-images.githubusercontent.com/32907141/163067982-f7b77eb0-59cd-4ffd-a2ee-ba5f2e1637c4.jpeg)

Not knx message on console

**Reply by ascillato** [reply]
And how you have configured your KNX button? If 3/0/14 is the physical address of the button it won't work. The button must send the command to a group address (different that its physical address) and that group address should be the same that you put in Tasmota. Also, your KNX Router must allow Multicast communications and your Wi-Fi Router too.

**Reply by detxm80** [reply]
3.0.14 is the group address not physical address. I’ve fritzbox 7590 and an enertex knx ip router, but with this configuration it doesn t work and no message on console

**Reply by ascillato** [reply]
1- Which device is your KNX button?
2- In your Tasmota configuration, in the part **_Group Addresses to receive data from_**, you should have only 1 trigger and as you have a button it should be `3/0/14 -> Toggle`. Also in the part **_Data to send to Group Addresses_**, it should be a different one than 3/0/14 so as to prevent a loop. In KNX the control GA must be different than the status GA.

**Reply by detxm80** [reply]
1- it is a simple btcino button linked to a makel 4 ch fold (i've variuos at home)
yes i configured it (toggle on off) into 3.0.14 group address. i've remove data to send to try. data to send set is obbligatory?
nothing change :(

**Reply by ascillato** [reply]
> data to send set is obbligatory?

No, none of them. You can use what you need. No problem with that.

> nothing change

With your latest changes, your Tasmota config is ok. So it seems that is an issue in your KNX router not routing the telegrams to Wi-Fi?

**Comment by detxm80** [comment]
https://www.enertex.de/e-katalog-securerouter.html

i've this one, it's compatible? and the fritz 7590?

i can view logs to track errors? where?

thanks for support adrian

**Comment by Noschvie** [comment]
Check the settings of your Fritzbox like „Enable IGMP Multicast Proxy“ oder „Enable multicast routing (IGMP Proxy)“

**Reply by detxm80** [reply]
> Check the settings of your Fritzbox like „Enable IGMP Multicast Proxy“ oder „Enable multicast routing (IGMP Proxy)“

there isn't this section on my fritz 7590.....

**Reply by detxm80** [reply]
fritzbox tell me that multicast is activated by default......
